content/applications/hr/lunch/orders.rst

```diff
@@ -0,0 +1,210 @@
+:show-content:
+
+========
+My Lunch
+========
+
+When the *Lunch* application is opened, the :guilabel:`Order Your Lunch` dashboard loads. This view
+is also accessed by navigating to :menuselection:`Lunch app --> My Lunch: New Order`.
+
+Order Your Lunch dashboard
+==========================
+
+On the main :guilabel:`Order Your Lunch` dashboard, all the necessary information needed to place an
+order is visible. The default filter for the products is :guilabel:`Available Today`. This filter
+shows only products that can be purchased that day.
```

Comment:
```suggestion
shows only products that can be purchased that day.
```

according to what? something an admin sets? availability in the inventory app?

Reply:
According to the vendors set days of availability- I updated it.